### ISSUE TITLE
Journal title `Künstliche Intelligenz` causes jabbrv to lock up

### ISSUE THREAD

**Issue by logological** [compholio/jabbrv#48]
jabbrv doesn't seem to like the journal title `Künstliche Intelligenz`; processing a bibliography entry with such a title causes pdflatex to lock up. Running pdflatex + biber + pdflatex with an up-to-date TeX Live 2024 on the following minimal example demonstrates the problem:

```
\documentclass{article}

\usepackage{biblatex}
\usepackage{jabbrv}
\begin{filecontents*}{\jobname.bib}
@article{sokol2020one,
  title={foo},
  author={bar},
  journal={Künstliche Intelligenz},
  volume={1},
  number={2},
  year={2020},
}
\end{filecontents*}

\addbibresource{\jobname.bib}
\begin{document}
\cite{sokol2020one}
\printbibliography
\end{document}
```

**Comment by compholio** [comment]
A lot of latex tools do not support unicode very well, have you tried to use the traditional typeset version?:
```
K\"unstliche Intelligenz
```

**Comment by logological** [comment]
Yes. The problem does not seem to have anything to do with the ü.  I see the same behaviour with `\"u`, `{\"u}`, etc.

**Comment by compholio** [comment]
Interesting, I didn't try with biber before (I can confirm that I see the same behavior with biber).  I'm not sure if this is a biber problem or a jabbrv problem, but I'll try to take a look later.
